client/js/components/StatsLeaderboard.vue

```diff
@@ -289,9 +292,25 @@ export default {
     tableRowClass(item) {
       return this.isCurrentPlayer(item.username) ? 'active-user-stats' : '';
     },
-    // Compute rank from total score
-    rank(totalScore) {
-      return this.totalScoresSorted.indexOf(totalScore) + 1;
+    /**
+     * @description Compute rank from total score and wins
+     * @returns the largest index (ranking) from players score and wins
+     *
+     * example:
+     * player1 - { score: 5, wins: 2 }
+     * player2 - { score: 5, wins: 4 }
+     * player3 - { score: 4, wins: 5 }
+     *
+     * all scores sorted - [5, 5, 4]
+     * all wins sorted - [5, 4, 2]
+     *
+     * player 2 will be ranked 1st, player 1 will be ranked 2nd, player 3 will be ranked 3rd
+     */
+    rank({ totalScore, totalWins }) {
+      const scoreIndex = this.totalScoresSorted.indexOf(totalScore);
+      const winsIndex = this.totalWinsSorted.indexOf(totalWins);
+      const ranking = Math.max(scoreIndex, winsIndex) + 1;
+      return ranking;
```

Comment:
So I think this addresses the change requested (breaking ties) but if I understand it correctly it makes other logical changes we want to avoid. Here are the intended requirements:

- Rankings should be determined first by points then by wins (total wins only matter to break a tie in points). Your change weighs both equally, but if one player gets 100 wins on week 1 and then never wins again, that should put them behind someone who took second place in that first week and then got first place every week after, even if the total wins are < 100 for that second player.

- If two players have the same number of total points, the tie should then be broken by the player with more total wins. If both players have the same total points and total wins, then they should have the same rank. So if two players tie for 4th place, they should both have a rank of 4 and the next player should have a rank of 6. This implementation would (I believe) give one of the players a rank of 5 somewhat randomly based on initial sort order

Reply:
Assuming that players scores will always be calculated correctly based on their winnings, this should theoretically work fine if a player's winning is 100 the first, that would mean their total points would also be very high as well.

I think I can make some improvements on how the data is mapped over, however. Let me provide another iteration on this. Thanks again for your input.

Reply:
I think there is a miscommunication about how scoring works which is driving the misalignment. Score is determined each week according to who gets the most wins that week:

1. 5 points for the player who gets the most wins that week
2. 4 points for the player who gets the 2nd-most wins that week
3. 3 points for the player who gets the 3rd-most wins that week
4. 1 point to all players who got at least one win but didn't place 1st-3rd

The total number of points (which determines rankings) is the sum of the points you've earned each week. That means that if a player wins 100 matches in the first week, they earn a maximum of 5 points for that week, whereas someone who wins 3 matches every week could also get 5 points per week, if they had the most wins on that week (even if it was less than 100)

Reply:
So the top-ranked player is the player who had the highest score from each week, not the one with the most total wins

Reply:
Okay cool, cool. Thanks for clarifying that. It makes much more sense now. I'll add that part to the README.md if it's not already there. Maybe we can break up the "How to Play" in a separate doc?

Reply:
> Okay cool, cool. Thanks for clarifying that. It makes much more sense now. I'll add that part to the README.md if it's not already there. Maybe we can break up the "How to Play" in a separate doc?

As in separate from the rest of the README? I'm open to that

Reply:
Yes, I'll add an issue for that.